php/commands/search-replace.php

```diff
@@ -115,17 +119,32 @@ public function __invoke( $args, $assoc_args ) {
 			$table->setRows( $report );
 			$table->display();
 
-			if ( !$dry_run )
+			if ( ! $dry_run ) {
 				WP_CLI::success( "Made $total replacements." );
+			}
 
 		}
 	}
 
-	private static function get_table_list( $args, $network, $all_with_prefix ) {
+	/**
+	 * Retrieve a list of tables from the database.
+	 *
+	 * @global wpdb $wpdb
+	 *
+	 * @param array $assoc_args Array of options passed to this command.
+	 *
+	 * @return array The array of table names.
+	 */
+	private static function get_table_list( $assoc_args ) {
```

Comment:
Can we make this argument `$limit_to` with the default behavior as `all-tables` and supporting `all-tables-with-prefix` or `network` as two possible options?